rs-matter/src/dm/networks/wireless.rs

```diff
@@ -154,10 +154,7 @@ where
     ///
     /// # Arguments
     /// - `buf`: The byte slice to store the state into
-    ///
-    /// Returns `Ok(None)` if the state has not changed, `Ok(Some(data))` if the state has changed
-    /// where `data` is the sub-slice of the buffer that contains the data to be persisted
-    pub fn store<'a>(&self, buf: &'a mut [u8]) -> Result<Option<&'a [u8]>, Error> {
+    pub fn store(&self, buf: &mut [u8]) -> Result<usize, Error> {
```

Comment:
while it's easy enough to guess what the returned usize is - I guess documenting that wouldn't hurt